numpy/core/tests/test_regression.py

```diff
@@ -2514,3 +2514,9 @@ def test_dot_big_stride(self):
         b[...] = 1
         assert b.strides[0] > int32_max * b.dtype.itemsize
         assert np.dot(b, b) == 2.0
+
+    def test_frompyfunc_name(self):
+        # name conversion was failing for python 3 strings
+        # resulting in the default '?' name.
+        f = np.frompyfunc(abs, 1, 1)
+        assert str(f) == "<ufunc 'abs (vectorized)'>"
```

Comment:
```suggestion
        def cassé(x):  # "broken" in french, testing unicode
            return x
        f = np.frompyfunc(cassé, 1, 1)
        assert str(f) == "<ufunc 'cassé (vectorized)'>"
```

Reply:
This won't work until the function is fixed.